torch/distributed/tensor/parallel/style.py

```diff
@@ -400,6 +400,23 @@ def __init__(
             assert len(self.input_kwarg_layouts) == len(self.desired_input_kwarg_layouts), \
                 "input_kwarg_layouts and desired_input_kwarg_layouts should have same length!"
 
+    def _prepare_input_arg(self, input: Any, mesh: DeviceMesh, input_layout: Placement, desired_layout: Placement):
```

Comment:
```suggestion
    def _prepare_input_arg(self,  input: Optional[Union[DTensor, torch.Tensor]], mesh: DeviceMesh, input_layout: Placement, desired_layout: Placement):
```

```

Reply:
How does this work if the module forward inputs contain objects other than `Tensor` / `DTensor`?

Reply:
Oh that's why this `input` type annotation is `Any`, since there are inputs that are not Tensor/DTensors, so `input` could be any type, and in this case we just simply return the input value